runtime/vm/createramclass.cpp

```diff
@@ -2475,6 +2475,12 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 					memcpy(ramClass->flattenedClassCache, flattenedClassCache, flattenedClassCacheAllocSize);
 				}
 #endif /* J9VM_OPT_VALHALLA_VALUE_TYPES */
+				ramClass->hotField1.hotFieldOffset =  UDATA_MAX;
+				ramClass->hotField1.hotFieldMethodHotness = 2; 
+				ramClass->hotField1.hotFieldThreshold = 200;
```

Comment:
Is 200 coming from OMR::Options::_hotFieldThreshold? Can we use the symbol, instead? Similarly for MethodHotness=2.